### PR TITLE
Add Serah kassim

### PR DESCRIPTION
**Pull request by ichtrojan**

### CODE REVIEW

**Comment by ichtrojan** [comment]
@Odusanya can you help with this too.

Thanks

File: logos/serah_kassim/serah_kassim.svg [hunk 1]
```diff
@@ -0,0 +1,369 @@
+<svg enable-background="new 0 0 1758 420" height="420" viewBox="0 0 1758 420" width="1758" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><image height="420" width="1758" xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAABt4AAAGkCAQAAAAcFZlsAAAABGdBTUEAALGPC/xhBQAAACBjSFJN
```
Comment (Odusanya) [inline logos/serah_kassim/serah_kassim.svg:1]: Hi @ichtrojan, this is a base 64 encoded PNG wrapped as an SVG. Kindly replace with a high-quality vector version of the logo.

**Review by Odusanya — CHANGES_REQUESTED** [review]

File: logos/serah_kassim/serah_kassim.svg (hunk 1 above)
Comment (ichtrojan) [reply logos/serah_kassim/serah_kassim.svg]: It's done

**Review by samijiyemi — APPROVED** [review]

File: logos/serah_kassim/serah_kassim.svg (hunk 1 above)
Comment (Odusanya) [reply logos/serah_kassim/serah_kassim.svg]: Hi @ichtrojan thanks for you're PR. [Same feedback](https://github.com/PaystackHQ/nigerialogos/pull/170#issuecomment-819511016) applies here

File: logos.json [hunk 2]
```diff
@@ -1341,6 +1341,21 @@
...
+		"category": [
+			"Financial Services",
+			"Software"
+		]
+	},
+	{
+		"title": "Serah Kassim",
+		"filename": "serah_kassim",
+		"url": "https://serahkassim.com/",
+		"category": ["fashion"]
```
Comment (Odusanya) [inline logos.json:1356]:
```suggestion
		"category": ["Fashion"]
```

Hi @ichtrojan sorry this fell off my radar. Kindly make this one last change so I can merge and take it live

**Review by Odusanya — CHANGES_REQUESTED** [review]

**Review by Odusanya — APPROVED** [review]

**Pull request merged**